python/tk_multi_about/context_browser.py

```diff
@@ -78,7 +84,13 @@ def process_result(self, result):
             i = self.add_item(browser_widget.ListItem)
             details = []
             details.append("<b>Project %s</b>" % d.get("name"))
-            details.append( d.get("sg_description") if d.get("sg_description") else "No Description" )            
+            details.append( d.get("sg_description") if d.get("sg_description") else "No Description" )
+
+            link_color = self._app.engine.style_constants.get("SG_HIGHLIGHT_COLOR","#18A7E3")
+            site_url = result.get("shotgun_url") if result.get("shotgun_url") else ""
+            site_str= "Site: <a href=\"{url}\" style=\"color:{color}\" >{url}</a> ".format(url=site_url,
+                                                                                           color=link_color)
+            details.append(site_str)
```

Comment:
One quick question. Is it clickable? It would be nice if we could.

Reply:
Ahh that's a really good point. I noticed that you could double-click it, but single click wouldn't work. However, I've now realised that the context item, in general, is double-clickable and that's what's providing the behaviour. I can have a look to see if there is anything we can do to make the link single click. I wonder if the view is masking the clicks?

Reply:
I don't know what `details` gets inserted into, but this might help. http://doc.qt.io/qt-4.8/qlabel.html#textInteractionFlags-prop

Reply:
And this http://doc.qt.io/qt-4.8/qlabel.html#openExternalLinks-prop

Reply:
That's how the `Forgot your password` link works on the login page.

Reply:
Great thats worked! thanks for that, I added the `setOpenExternalLinks` to the widget and it works. However `tk-framework-widget` controls the label on the item.
I could either make a change to the frame work, or I since I am derived from the `BrowserWidget` I can do this instead:

```
    def add_item(self, item_class):
        """
        Adds a list item. Returns the created object.
        """
        widget = super(ContextBrowserWidget, self).add_item(item_class)
        widget.ui.details.setOpenExternalLinks(True)

        return widget
```

Which would be better?

Reply:
I think `ui`property being public was not the intent. Maybe we should fix the BrowserWidget item to that hyperlinks are always followed. What do you think @manneohrstrom ?

Reply:
yeah I agree that probably putting it in the framework is the better option, I doubt its likely to cause issues elsewhere, but not certain, having never used that setting before.

Reply:
`tk-framework-widget` is old and deprecated - let's not spend cycles making changes there. That effort is better spent fixing other bugs!